SCons/Tool/MSCommon/common.py

```diff
@@ -342,6 +342,18 @@ def normalize_env(env, keys, force: bool=False):
     if sys32_wbem_dir not in normenv['PATH']:
         normenv['PATH'] = normenv['PATH'] + os.pathsep + sys32_wbem_dir
 
+    # ProgramFiles for PowerShell 7 Path and PSModulePath
+    progfiles_dir = os.environ.get("ProgramFiles")
+    if not progfiles_dir:
+        sysroot_drive, _ = os.path.splitdrive(sys32_dir)
+        sysroot_path = sysroot_drive + os.sep
+        progfiles_dir = os.path.join(sysroot_path, "Program Files")
+
+    # Powershell 7
+    progfiles_ps_dir = os.path.join(progfiles_dir, r"PowerShell\7")
```

Comment:
why r"PowerShell\7"? is that a single path element and not PowerShell \ 7 ?

Reply:
raw string it is `Powershell\7`:

os.path.join(os.environ["ProgramFiles"], r"Powershell\7")
'C:\\Program Files\\Powershell\\7'

Reply:
Would you prefer `os.path.join(progfiles_dir, "PowerShell", "7")`?